src/main.rs

```diff
@@ -37,9 +40,53 @@ use codec::{FtpCodec, StringCodec};
 use ftp::{Answer, ResultCode};
 
 const DEFAULT_PORT: u16 = 4321;
+const MONTHS: [&'static str; 12] = ["Jan", "Feb", "Mar", "Apr", "May", "Jun",
+                                     "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
 
 type Writer = SplitSink<Framed<TcpStream, FtpCodec>>;
 
+cfg_if! {
```

Comment:
What is the advantage of using that vs `#[cfg(windows)]` and `#[cfg(not(windows))]`?

Reply:
Not much. I just really like this macro. :)